src/conversions/std/slice.rs

```diff
@@ -20,21 +18,8 @@ impl<'a> IntoPy<PyObject> for &'a [u8] {
     }
 }
 
-#[cfg(feature = "gil-refs")]
-impl<'py> crate::FromPyObject<'py> for &'py [u8] {
-    fn extract(obj: &'py PyAny) -> PyResult<Self> {
-        Ok(obj.downcast::<PyBytes>()?.as_bytes())
-    }
-
-    #[cfg(feature = "experimental-inspect")]
-    fn type_input() -> TypeInfo {
-        Self::type_output()
-    }
-}
-
```

Comment:
I think it might be good to keep the conditional implementations here, only because code which was previously calling `FromPyObject::extract(obj)` using UFCS will break when removing these implementations. So the action of removing the `gil-refs` feature seems like a gentler break point IMO.

Reply:
Ah, that makes sense, haven't thought of that. I'll drop the comment removing those